recipes-kernel/linux/linux-qoriq/0001-perf-Make-perf-able-to-build-with-latest-libbfd.patch

```diff
@@ -0,0 +1,62 @@
+From c27ccf94fb57077a420abe93da15ebe0ad6ce34c Mon Sep 17 00:00:00 2001
+From: Lei Maohui <leimaohui@cn.fujitsu.com>
+Date: Wed, 29 Apr 2020 09:52:27 +0900
+Subject: [PATCH] perf: Make perf able to build with latest libbfd
+
+libbfd has changed the bfd_section_* macros to inline functions
+bfd_section_<field> since 2019-09-18. See below two commits:
+  o http://www.sourceware.org/ml/gdb-cvs/2019-09/msg00064.html
+  o https://www.sourceware.org/ml/gdb-cvs/2019-09/msg00072.html
+
+This fix make perf able to build with both old and new libbfd.
+
```

Comment:
It's a bit poor as description from my point of view: https://www.kernel.org/doc/html/latest/process/submitting-patches.html
From technical perspective - cool, but I'd also like to know why it's not on LKML, is there another upstream discussion we don't know about...

Reply:
Yes, this patch is not an original one. I've introduced the cherry-picked version in `linux-imx-5.4.3`, which could be tracked on LKML [1].

@leimaohui: this was exactly my point above - if there is a need to include this patch for `linux-qoriq` - then the original version should be taken.

[1]: https://lkml.org/lkml/2020/1/28/1074

Reply:
@zandrey: good to know. Maybe recipes-kernel/linux/linux-imx-5.4.3/0001-perf-Make-perf-able-to-build-with-latest-libbfd.patch should be regenerated in that case (as extra bonus) but I have to look into linux-imx-5.4.3